### ISSUE TITLE
0.8.2 was never tagged

### ISSUE THREAD

**Issue by thinkl33t** [smarkets/marge-bot#215]
The 0.8.2 release was never tagged, meaning the CI hasn't released a docker image etc.

```
git checkout 7df5003bbe06b61b6429b5defb24fc1d6b7b5406 && git tag -l '0.8.2' && git push origin '0.8.2'
```

By someone with commit access should do it i think?

**Comment by JaimeLennox** [comment]
Hmm, it seems 0.8.2 included a lot more than is in the changelog as well. I'm included to just remove it and release a 0.9.0.

**Comment by JaimeLennox** [comment]
Superceded by 0.9.0 (#216)
